db/app.js

```diff
@@ -1,11 +1,11 @@
 const express = require("express");
```

Comment:
your app.js, model.js and controller.js modules should be located in your 'root' directory; i.e The main folder of the repo.